src/components/user/schedule/Schedule.jsx

```diff
@@ -10,10 +10,20 @@ const Schedule = ({ eventList }) => {
   const [filteredEvents, setFilteredEvents] = useState(eventList);
 
   const filterChange = (filterType) => {
-    if (filterType === "all") {
-      setFilteredEvents(events);
+    if (filterType === "hackweek") {
+      setFilteredEvents(
+        events.filter((event) => event.description.includes("#workshop")),
+      );
+    } else if (filterType === "hackathon") {
+      setFilteredEvents(
+        events.filter(
+          (event) =>
+            event.description.includes("#general") ||
```

Comment:
there can be workshops during the hackathon as well, the filtering should be based on the days currently being looked at, so for hackathon it should only be Saturday and sunday, while hackweek looks at mon - fri